devops/jobs/CreateSandboxCI.groovy

```diff
@@ -106,6 +106,20 @@ class CreateSandboxCI {
                   booleanParam("journals",false)
                   booleanParam("video_pipeline",false)
                   booleanParam("video_encode_worker",false)
+                } else if (type == 'ProspectusBiHourly') {
+                  predefinedProp('dns_name','prospectus-bihourly')
+                  predefinedProp('name_tag','prospectus-bihourly')
+
+                  predefinedProp('server_type', 'full_edx_installation_from_scratch')
+                  booleanParam('reconfigure',true)
+                  booleanParam('recreate', true)
```

Comment:
the first and last here are the default. The middle won't matter if you take the default for the first.

Defaults are here: https://github.com/edx/jenkins-job-dsl/blob/029b9c6a920f1971be69b4f480c6da8de547a185/devops/jobs/CreateSandbox.groovy
To see what reconfigure does see: https://github.com/edx/configuration/blob/25850cf1de16066da714551c302fb00c8d418b24/util/jenkins/ansible-provision.sh#L489 + like 10 or so lines. (The setting has been kind of redundant for a while, since it's basically always superseded by server_type.)